### PR TITLE
Next/Prev pagination

### PR DESCRIPTION
**Pull request by buberdds**
In MVP we won't have number of items returned by API so in details pages we will use different pagiantion.

last page for 21 txes
Screenshot 2023-02-08 at 16 17 13

### CODE REVIEW

File: src/app/components/Table/TablePagination.tsx [hunk 1]
```diff
@@ -3,33 +3,59 @@ import PaginationItem from '@mui/material/PaginationItem'
...
+  linkToPage,
+}) => {
   const { t } = useTranslation()
 
   return (
     <Pagination
-      count={numberOfPages}
+      boundaryCount={prevNextOnly ? 0 : undefined}
+      siblingCount={prevNextOnly ? negatedSiblingCount : undefined}
+      count={prevNextOnly ? (hasMore ? numberOfPages : selectedPage) : numberOfPages}
```
Comment (lukaw3d) [inline src/app/components/Table/TablePagination.tsx:33]: All of these customizations are making me think maybe we should
```tsx
if (prevNextOnly) return <OurPrevNextTwoButtonsImplementation>
```

File: src/app/components/Table/TablePagination.tsx [hunk 2]
```diff
@@ -3,33 +3,59 @@ import PaginationItem from '@mui/material/PaginationItem'
...
         <PaginationItem
+          hidden={prevNextOnly && ['end-ellipsis', 'start-ellipsis'].includes(item.type)}
           slots={{
             first: () => <>{t('pagination.first')}</>,
             last: () => <>{t('pagination.last')}</>,
+            previous: () => (
+              <>
+                <NavigateBeforeIcon fontSize="small" /> {t('pagination.previous')}
+              </>
+            ),
```
Comment (lukaw3d) [inline src/app/components/Table/TablePagination.tsx:45]: Is the label change intentional on all tables?

| Before | After |
| --- | --- |
|  oasis-explorer-frontend-19af130 pages dev_emerald_blocks  | localhost_1234_emerald_transactions  |

Comment (buberdds) [reply src/app/components/Table/TablePagination.tsx:45]: yes it is, I was asking UX if we want to unify next prev labels

File: src/app/pages/AccountDetailsPage/TransactionsCard.tsx [hunk 3]
```diff
@@ -15,7 +15,7 @@ export const TransactionsCard: FC = () => {
   const txsPagination = useSearchParamsPagination('page')
   const txsOffset = (txsPagination.selectedPage - 1) * NUMBER_OF_ITEMS_ON_SEPARATE_PAGE
   const transactionsQuery = useGetEmeraldTransactions({
-    limit: NUMBER_OF_ITEMS_ON_SEPARATE_PAGE,
+    limit: NUMBER_OF_ITEMS_WITH_NEXT_PAGE_VALIDATION,
```
[the remark this answers is not in the document]
Comment (buberdds) [reply src/app/pages/AccountDetailsPage/TransactionsCard.tsx:18]: Thanks! will check out hooks later today. I am not huge fan of prefetching pages idea.

**Pull request closed without merging**